packages/ui-tree-browser/src/TreeBrowser/props.ts

```diff
@@ -52,6 +52,7 @@ type TreeBrowserOwnProps = {
   onItemClick?: (...args: any[]) => any
   treeLabel?: string
   renderContent?: (...args: any[]) => any
+  sortOrder?: (arg1: any, arg2: any) => number
```

Comment:
I would name these arguments `obj1` and `obj2` here as well, like you did in the defaultProps. That is more consistent, and indicates that these are objects.